packages/react/src/core/ContextMenu.tsx

```diff
@@ -54,7 +56,7 @@ export function ContextMenu({
   const [state, setState] = useState<'idle' | 'add_file' | 'add_folder' | 'add_failed'>('idle');
   const inputRef = useRef<HTMLInputElement>(null);
 
-  if (!onFileChange) {
+  if (!onFileChange || !allowEditPatterns?.length) {
```

Comment:
```suggestion
  if (!allowEditPatterns?.length) {
```

(so that the context menu still shows up which is useful when playing around with the component)